sdk/eventgrid/eventgrid/README.md

```diff
@@ -9,7 +9,7 @@ Use the client library to:
 - Generate Shared Access Signatures for Event Grid topics
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/) |
-[Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid/v/next) |
 [API reference documentation](https://aka.ms/azsdk-js-eventgrid-ref-docs) |
```

Comment:
Who do we talk to for making https://aka.ms/azsdk-js-eventgrid-ref-docs point to https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-eventgrid/3.0.0-beta.2/index.html until the reference docs in docs.microsoft.com are lit up?

Reply:
Our team has control over that link, I just updated to point at the above (it had been pointing at https://azure.github.io/azure-sdk-for-js/eventgrid.html).